src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+  address: Address,
+  phone: string,
+  website: string,
+  company: Company,
+}
+
+export interface Address {
+  street: string,
+  suite: string,
+  city: string,
+  zipcode: string,
+  geo: Geo,
+}
+
+export interface Geo {
+  lat: string,
+  lng: string,
+}
+
+export interface Company {
+  name: string,
+  catchPhrase: string,
+  bs: string,
+}
+
+export interface Post {
+  userId: number,
```

Comment:
If it is a user id that is related to User interface, better to create a more appropriate type

 ```suggestion
  userId: User['id'],
```